app/models/outgoing_sms_communication.rb

```diff
@@ -0,0 +1,28 @@
+class OutgoingSmsCommunication < ApplicationRecord
+  belongs_to :user
+
+  enum message_type: [
+    :initial_message,
+    :cancellation_message,
+    :reminder_message
+  ]
+
+  def self.record(recip_email, status, type)
+    user = User.find_by(email: recip_email)
+    msg_type = type_map[type]
+
+    create(
+      user: user,
+      status: status,
+      message_type: msg_type
+    )
```

Comment:
Per our discussion about this while reviewing live, I think it would be good to re-write this as `self.create`, modify the arguments per the above comment, and then call `super` and pass it the modified arguments which can be sent to the superclass' `create` method. We said we thought this would make the code more readable for later contributors.

Reply:
Done.